api/v1beta2/sparkapplication_types.go

```diff
@@ -639,22 +576,22 @@ type SecretType string
 
 // An enumeration of secret types supported.
 const (
-	// GCPServiceAccountSecret is for secrets from a GCP service account Json key file that needs
+	// SecretTypeGCPServiceAccount is for secrets from a GCP service account Json key file that needs
```

Comment:
General question for maintainers-- Do we want to target this as an area for separate enhancement, and tie less exclusively to GCP?

Reply:
The API definition is the same as before, just do some renaming with the variables.